.github/workflows/ci-cd.yml

```diff
@@ -106,7 +106,6 @@ jobs:
           context: .
           file: ./Dockerfile
           platforms: linux/amd64,linux/arm64
-          push: ${{ github.event_name != 'pull_request' && steps.prep.outputs.tags != 'fluree/ledger:none' }}
```

Comment:
I don't think you want to delete this entire line. I believe that will disable pushing altogether, or if it does default to true, we still want to filter out that `fluree/ledger:none` tag.

I was just suggesting removing the `github.event_name != 'pull_request'` part.